<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>851</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>L/COL/PIL</string>
     </attribute>
     <widget class="QTableWidget" name="tw_mostrarCola">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>50</y>
        <width>271</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_mostrarPila">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>271</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_elimiar">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>-40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar final</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delPenultimo">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>210</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar penultimo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delCrec">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>240</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Creciente</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delI">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>90</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar inicio</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_mostrar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>311</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_in4ta">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>210</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar 4ta Pos</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_del2">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar 2da pos</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sp_pos">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>270</y>
        <width>46</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sp_del">
      <property name="geometry">
       <rect>
        <x>1230</x>
        <y>300</y>
        <width>46</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inCrec">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>300</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Creciente</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_in3ra">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>180</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar 3ra Pos</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delPos">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>300</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Pos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ingresar">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>440</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Proceder</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delPila">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>400</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Pila</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inicio">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>100</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Inicio</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delCola">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>400</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Cola</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delAnt">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>180</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Antepenultimo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inPos">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>270</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Posicion</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inCola">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>370</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Cola</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inFin">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>120</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Final</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_ante">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>-40</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Antepenultima</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_in4ta_2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>240</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Penultima</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_del3">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>150</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar 3ra pos</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inPila">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>370</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Pila</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_in2nd">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>150</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar 2da Pos</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inCrec_2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>330</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar Decreciente</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delDec">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>270</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Decreciente</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tbx_letra">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>40</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>0</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>INGRESE LETRA:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>DOBLE</string>
     </attribute>
     <widget class="QLabel" name="lbl1_2">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>INGRESE LETRA:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tbx_textD">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_delfind">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>130</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar final</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_inantd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>180</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_doble">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>250</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Porceder</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_doble">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>521</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sp">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>240</y>
        <width>46</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>TDA</string>
     </attribute>
     <widget class="QTableWidget" name="TW_MOSTRAR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>921</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="GB_Menu">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>20</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Menu</string>
      </property>
      <widget class="QRadioButton" name="RB_Lista">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Lista</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Pila">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Pila</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Cola">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cola</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Arbol">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Árbol</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_Sub_Menu">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>30</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Sub_Menu</string>
      </property>
      <widget class="QRadioButton" name="RB_Simple">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Simple</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Doble">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Doble</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="TB_Letra">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>300</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_Aceptar">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>340</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>INGRESAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="Lbl_Letra">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>300</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Letra:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CB_ORDEN">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>150</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>PUSH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>POP</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="GB_orden">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>190</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Tipo de orden</string>
      </property>
      <widget class="QRadioButton" name="rb_post">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>97</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>PostOrden</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_inorden">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>97</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Inorden</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_preorden">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>97</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Preorden</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1314</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
